move/enclave/README.md

```diff
@@ -0,0 +1,21 @@
+# Nautilus Package
+
+Nautilus is a framework for **secure and verifiable off-chain computation on Sui**. For full product details, see the [Nautilus documentation](https://docs.sui.io/concepts/cryptography/nautilus).
+
+The Nautilus package includes a pattern to manage enclaves onchain with creating and updating enclave configs, registering enclaves and verifying signature with it. 
+
+## Published Envs
+
+The Nautilus package is available in Testnet and Mainnet. To use it, add the following to your `Move.toml`:
+
+For testnet:
+```toml
+[dependencies]
+nautilus = { git = "https://github.com/MystenLabs/nautilus.git", subdir = "nautilus", rev = "testnet" }
```

Comment:
checking @tonylee08 this is ok if the directory "move/enclave" is not called "nautilus" right? and is this the right git path?

Reply:
removed since its outdated!